src/components/global-toolbar/index.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { toggleSettingsModal, toggleTheme } from '../../actions';
+import ExperimentsIcon from '../icons/experiments';
+import IconButton from '../icon-button';
+import LogoIcon from '../icons/logo';
+import SettingsIcon from '../icons/settings';
+import ThemeIcon from '../icons/theme';
+import TreeIcon from '../icons/tree';
+
+import './global-toolbar.css';
+
+/**
+ * Main controls for filtering the chart data
+ * @param {Function} onToggleTheme Handle toggling theme between light/dark
+ * @param {string} theme Kedro UI light/dark theme
+ */
+export const GlobalToolbar = ({
+  onToggleSettingsModal,
+  onToggleTheme,
+  theme,
+  visible,
+}) => {
+  return (
+    <>
+      <div className="pipeline-global-toolbar">
+        <ul className="pipeline-global-routes-toolbar kedro">
+          <IconButton
+            ariaLabel={'Kedro Viz logo and link'}
+            className={'pipeline-menu-button--logo pipeline-menu-button--large'}
+            disabled={false}
+            icon={LogoIcon}
+          />
+          <IconButton
+            ariaLabel={'View your pipeline'}
+            className={'pipeline-menu-button--large'}
+            disabled={false}
+            icon={TreeIcon}
+            labelText={'Pipeline'}
+          />
+          <IconButton
+            ariaLabel={'View your experiments'}
+            className={'pipeline-menu-button--large'}
+            disabled={false}
+            icon={ExperimentsIcon}
+            labelText={'Experiments tracking'}
+          />
+        </ul>
+        <ul className="pipeline-global-control-toolbar kedro">
+          <IconButton
+            ariaLive="polite"
+            ariaLabel={`Change to ${
+              theme === 'light' ? 'dark' : 'light'
+            } theme`}
+            className={
+              'pipeline-menu-button--theme pipeline-menu-button--large'
+            }
+            onClick={() => onToggleTheme(theme === 'light' ? 'dark' : 'light')}
+            icon={ThemeIcon}
+            labelText="Toggle theme"
+            visible={visible.themeBtn}
```

Comment:
Same as my previous comment, I don't think we need the `visible` prop here given that we don't have any use case for the themeBtn to be false 😆